/jds/bin/diff -uprN gnome-panel-2.26.3.old/gnome-panel/gnome-desktop-item-edit.c gnome-panel-2.26.3/gnome-panel/gnome-desktop-item-edit.c
--- gnome-panel-2.26.3.old/gnome-panel/gnome-desktop-item-edit.c	2009-07-01 20:36:42.760796000 +0100
+++ gnome-panel-2.26.3/gnome-panel/gnome-desktop-item-edit.c	2009-07-01 20:43:32.860614000 +0100
@@ -17,7 +17,6 @@
 #include "applet.h"
 GSList *panel_applet_list_applets (void) { return NULL; }
 #include "panel-gconf.h"
-GConfClient *panel_gconf_get_client (void) { return NULL; }
 #include "panel-config-global.h"
 gboolean panel_global_config_get_tooltips_enabled (void) { return FALSE; }
 #include "panel-lockdown.h"
